src/domUpdates.js

```diff
@@ -81,12 +80,12 @@ let domUpdates = {
     pantry.forEach(ingInPantry => {
       pantryArea.innerHTML += `<article class="single-ingredient-card">
     <article class="ingredient-name-area">
-      <h2 class="ingredient-name">${ingInPantry.name}</h2>
+      <h2 class="ingredient-name" tabindex="0">${ingInPantry.name}</h2>
     </article>
     <article class="ingredient-quantity-section">
-      <button>-</button>
-      <p class = 'ingredient-quantity'>${ingInPantry.amount}</p>
-      <button>+</button>
+      <button alt="decrement ingredient quantity button">-</button>
```

Comment:
thanks for adding those `alt` tags for the buttons, we were definitely overlooking that aspect.